code/modules/tgui/modules/ert_manager.dm

```diff
@@ -104,5 +108,21 @@
 			log_admin("[key_name(usr)] dispatched a [ert_type] ERT. Slots: [slot_text]")
 			GLOB.event_announcement.Announce("Attention, [station_name()]. We are attempting to assemble an ERT. Standby.", "ERT Protocol Activated")
 			trigger_armed_response_team(D, commander_slots, security_slots, medical_slots, engineering_slots, janitor_slots, paranormal_slots, cyborg_slots, cyborg_security)
+		if("dispatch_ds")
+			safety = !(safety)
+			var/datum/response_team/D
+			D = new /datum/response_team/epsilon
+			var/slots_list = list()
+			if(commander_slots > 0)
+				slots_list += "commander: [commander_slots]"
+			if(security_slots > 0)
+				slots_list += "security: [security_slots]"
+			if(cyborg_slots > 0)
+				slots_list += "cyborg: [cyborg_slots]"
+			var/slot_text = english_list(slots_list)
+			notify_ghosts("A Deathsquad is being dispatched. Open positions: [slot_text]")
+			message_admins("[key_name_admin(usr)] dispatched a Deathsquad. Slots: [slot_text]", 1)
+			log_admin("[key_name(usr)] dispatched a Deathsquad. Slots: [slot_text]")
+			trigger_armed_response_team(D, commander_slots, security_slots, cyborg_slots)
```

Comment:
Do not integrate DS into the ERT panel. The extra slots don't make sense and they need custom spawning logic.

Reply:
DS being TGUI'd however is something good. The current methods to spawn them are a pile of crap and prone to breaking without informing anyone of anything. This does seem like the closest currently existing TGUI menu to include them in unless we want a dedicated menu

Reply:
Thats not my point. 

The current DS system is designed in a very particular way, because the candidates are designed to be chosen, not randompicked, that way admins can select people who will actually be capable of ending the round as required. The current system is fine, so long as you're not mob swapping, which people have gotten used to at this point.

Also making this a TGUI wouldn't help the mob swapping issue, as those freeze when you swap mobs as well with aghost, even with the transfer_uis proc.

Reply:
Fair points regarding candidate picking and TGUI, however

> The current system is fine, so long as you're not mob swapping, which people have gotten used to at this point.

This seems to persistently be an issue even now, as each individual admin sends a DS reasonably infrequently they forget by the time they next send one.
The amount of times I've found myself having to asay "dont freaking ghost if you are gonna call a DS though" only for someone to do that and say "I didn't know" is more than it should be.
Something needs to be done so it's not so garbo, TGUI or not.

Reply:
DS now has its own TGUI, cleaning up the code is still in progress